src/language/lang_hu.php

```diff
@@ -0,0 +1,333 @@
+	'If you have any questions about what is or is not allowed, please ask an organizer.' => 'Ha kérdésed merülne föl, hogy mit szabad és mit nem, kérdezz meg egy szervezőt.',
+	'Have fun!' => 'Jó szórakozást!',
+	'Name' => 'Név',
+	'Email' => 'Email',
+	'Token' => 'Token',
+	'Team Registration' => 'Csapat regisztráció',
+	'Register to play Capture The Flag here. Once you have registered, you will be logged in.' => 'Regisztrálj a Capture The Flag versenyre! A regisztráció után a rendszer automatikusan beléptet.',
+	'Team Name' => 'Csapatnév',
+	'Password' => 'Jelszó',
+	'Choose an Emblem' => 'Válassz egy logót',
+	'Sign Up' => 'Regisztráció',
+	'Not Available' => 'Nem elérhető',
+	'Team Registration will be open soon, stay tuned!' => 'A regisztráció hamarosan megnyílik, türelem.',
+	'Try Again' => 'Próbáld újra',
+	'Select' => 'Válassz',
+	'Team Login' => 'Csapat bejelentkezés',
+	'Please login here. If you have not registered, you may do so by clicking "Sign Up" below. ' => 'Jelentkezz be itt. Ha még nem regisztráltál, kattints a "Regisztráció" gombra alul. ',
+	'Team Login will be open soon, stay tuned!' => 'A belépés hamarosan megnyílik, türelem.',
+	'ERROR' => 'HIBA',
+	'Start Over' => 'Kezdd újra',
+	'Window is too small' => 'Az ablak túl kicsi',
+	'For the best CTF experience, please make window size bigger.' => 'A jobb élmény érdekében növeld az ablak méretét.',
+	'Thank you.' => 'Köszönjük.',
+	'Logout' => 'Kijelentkezés',
+	'Registration' => 'Regisztráció',
+	'Play CTF' => 'Kezdjük',
+	'Rules' => 'Szabályok',
+
+//Translations for GameboardController
+	'Admin' => 'Admin',
+	'ADMIN' => 'ADMIN',
+	'Navigation' => 'Navigáció',
+	'View Mode' => 'Néző mód',
+	'View mode' => 'Néző mód',
+	'Tutorial' => 'Hogy működik',
+	'Scoreboard' => 'Eredményjelző',
+	'You' => 'Ti',
+	'Others' => 'Többiek',
+	'All' => 'Mind',
+	'Leaderboard' => 'Ranglista',
+	'Announcements' => 'Közlemények',
+	'Teams' => 'Csapatok',
+	'Filter' => 'Szűrő',
+	'Activity' => 'Történések',
+	'Game Clock' => 'Visszaszámláló',
+
+//Translations for AdminController
+	'Auto' => 'Véletlenszerű',
```

Comment:
Thank you for all your feedback. I'm not really sure about this last one, since it will appear when the admin selects the country for a new level:
![selection_003](https://cloud.githubusercontent.com/assets/12611720/15725882/428cd66a-284e-11e6-92f6-a28081dbe466.jpg)
And in this context I believe "Véletlenszerű" makes more sense. But "Automatikus" is okay too, so if you insist, I will change :)